Adding new ppl profiles (#558)
usds/website: _people/0750_fatima-noor.md

---
# Page template info (DO NOT EDIT)
layout: default
banner_file: banner--people-projects-lg.svg
banner_file_mobile: banner--people-projects-sm.svg
people_page: true

# Page link (Edit this)
permalink: people/fatima-noor

# People Carousel and Page (Edit this)
name: Fatima Noor
pronouns: She/Her
alumni: false
cop: Product
city_of_origin: Memphis, TN
previously_at: White House Domestic Policy Council and Department of Homeland Security
image_name: fatima-noor.jpg
quote: "I love that we prioritize delivering the most meaningful impact to communities that need it the most."

---

**What’s your background?**

> I was born in Somalia and know firsthand what it’s like to be a refugee. The experience I had growing up in different places from Kenya to Europe, and eventually the United States, influenced my decision to take a job working with new immigrants and future Americans just arriving in the US.

Reviewer: ```suggestion
I was born in Somalia and know firsthand what it’s like to be a refugee. The experiences I had growing up in Kenya, Europe, and eventually the United States, influenced my decision to take a job working with new immigrants and future Americans just arriving in the U.S.
```

> I enjoyed the work very much and wanted to scale my impact- leading me to join the Department of Homeland Security where I was assigned to the White House Domestic Policy Council to work on several federal immigration policies. I led the Taskforce on New Americans focused on the social and economic integration of our newest citizens and ran a national campaign aimed at providing assistance to local governments building welcoming communities for refugees.

Reviewer: ```suggestion
I enjoyed the work very much and wanted to scale my impact, leading me to join the Department of Homeland Security, where I was assigned to the White House Domestic Policy Council to work on several federal immigration policies. I led the Taskforce on New Americans, focused on the social and economic integration of our newest citizens, and ran a national campaign aimed at providing assistance to local governments building welcoming communities for refugees.
```

**What inspired you to join USDS?**

During my time on the WH Domestic Policy Council, I became aware of a handful of folks in my neighboring office wearing hoodies and jeans on the otherwise very formally dressed Presidential campus. I soon learned about the U.S. Digital Service and their work to digitize the naturalization process for new Americans.
When I learned more about the mission, I jumped at the opportunity to continue serving the American public in an area I was passionate about.
Since USDS is part of the White House, they have a lot of convening power to bring agencies together to harness and scale repeatable best practices across the federal government.
Our mission and the grounding thing we often say is that we’re doing the “greatest good for greatest number of people, in the greatest need.” I love that we prioritize delivering the most meaningful impact to communities that need it the most. Our explicit focus on how we can help actual human beings is at the core of what influenced me to join USDS.

**How does your work make an impact?**

> Whenever we think of government services, there are regulations, policy, compliance as the backbone of these services. Not only is modernizing the technology important but understanding how to modernize the policy is also critical.

Reviewer: ```suggestion
When we think of government services, regulations, policy, and compliance are the backbone of those services. Not only is modernizing the technology important, but it's also critical to understand how to modernize the policy.
```

> USDS really understands the need to combine these two facets of a government service — the spirit and the implementation of the service — to build usable, modern solutions that serve all Americans.

Reviewer: ```suggestion
USDS really understands the need to combine these two facets of a government service — both the spirit and the implementation — to build usable, modern solutions that serve all Americans.
```

This has led to the work I’m currently doing at the Office of Personnel Management (OPM) to improve the experience of public servants across the federal workforce, as well as the millions of customers that rely on OPM’s services. As we are building towards a 21st century government, we must have an employee experience that is on-par with what the private sector has been delivering for decades.

**What will you miss most about USDS when you leave?**

The people! Without a question, the talented and awesome humans I get to call my colleagues are the best thing about USDS!

> It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenging themselves to think about the end user — the Veteran or the immigrant or the federal employee — relying on these services.

Reviewer: ```suggestion
It’s inspiring being around a group of people who are not afraid to dive into complex problems and constantly challenge themselves to think about the end user — the Veteran or the immigrant or the federal employee — who rely on these services.
```

> USDSers are the most empathetic group of people I’ve ever come across. Regardless of what problem we’re trying to solve, we always put ourselves in the shoes of the users and the people who are going to be impacted by our work.

Reviewer: ```suggestion
USDSers are the most empathetic group of people I’ve ever come across. Regardless of the problem we’re trying to solve, we always put ourselves in the shoes of the users and the people who will be impacted by our work.
```

> Not only are most people who join USDS driven by the mission, there’s a continuous challenge to ensure that our work truly addresses tangible human need. We constantly question the solutions we design and ensure they truly meet the needs of the people that we’re delivering for.

Reviewer: ```suggestion
Not only are most people who join USDS driven by the mission, there’s a continuous challenge to ensure that our work truly addresses tangible human need. We constantly question the solutions we design and ensure they truly meet the needs of the people that we’re serving.
```
